src/actix/api/service_api.rs

```diff
@@ -80,9 +79,8 @@ async fn metrics(
         .prepare_data(
             &access,
             TelemetryDetail {
-                level: DetailsLevel::Level3,
+                level: DetailsLevel::Level4,
```

Comment:
This will make the call to `/metrics` more expensive, but I guess we don't really have a choice.